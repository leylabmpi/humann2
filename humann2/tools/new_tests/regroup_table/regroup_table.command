humann2_regroup_table \
	     -i regroup_table.input \
	     -L M2 \
	     -o regroup_table.output \
	     -c custom_groups.input \