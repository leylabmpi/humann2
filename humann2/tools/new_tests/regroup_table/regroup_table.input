#	S1	S2	S3	S4
M1	-	-	-	-
M2	-	-	-	-
UNMAPPED	5	10	15	20
F1	2	2	2	0
F1|S1	1	1	1	0
F1|S2	1	1	1	0
F2	2	0	0	0
F2|S1	1	0	0	0
F2|S2	1	0	0	0
F3	4	4	4	4
F3|S1	1.5	1.5	1.5	1.5
F3|S2	2.5	2.5	2.5	2.5
