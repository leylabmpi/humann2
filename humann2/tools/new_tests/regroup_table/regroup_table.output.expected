# SAMPLES	S1	S2	S3	S4
M1	-	-	-	-
M2	-	-	-	-
UNMAPPED	5	10	15	20
UNGROUPED	4	4	4	4
UNGROUPED|S1	1.5	1.5	1.5	1.5
UNGROUPED|S2	2.5	2.5	2.5	2.5
Ga	4	2	2	0
Ga|S1	2	1	1	0
Ga|S2	2	1	1	0
Gb	2	2	2	0
Gb|S1	1	1	1	0
Gb|S2	1	1	1	0
